src/Bicep.Cli/Arguments/BuildArguments.cs

```diff
@@ -101,6 +114,8 @@ public BuildArguments(string[] args) : base(Constants.Command.Build)
 
         public string? OutputFile { get; }
 
+        public string? DiagnosticsFormat { get; }
```

Comment:
>string [](http://example.com/codeflow?start=15&length=6)

Since we have a limited set of formats that we support, should we use an enum instead of a string?

Reply:
The other benefit may also be that we can validate the format earlier before any diagnostics have to be emitted.

Reply:
Updated it to be an enum here.